docs/source/pythonapi/examples/mgxs-part-iv.ipynb

```diff
@@ -952,12 +952,13 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "We will now use the `Library` to produce a multi-group cross section data set for use by the OpenMC multi-group solver.  "
+    "We will now use the `Library` to produce a multi-group cross section data set for use by the OpenMC multi-group solver.  \n",
+    "Note that since we have ran so few histories, is not unreasonable to expect some divisions by zero errors.  This will show up as a runtime warning in the following step."
```

Comment:
Note that these errors divide-by-zero warnings exist in the other MGXS notebook I as well. No mention is made of these errors, though I agree that it may be worthwhile to point it out as you do here.